### PR TITLE
speedtest: add page

### PR DESCRIPTION
**Pull request by marchersimon**
speedtest: add page

### CODE REVIEW

File: pages/linux/speedtest.md
Comment (bl-ue) [inline pages/linux/speedtest.md:3]:
```suggestion
> Test internet bandwidth using https://www.speedtest.net.
```

File: pages/linux/speedtest.md
Comment (bl-ue) [inline pages/linux/speedtest.md:5]:
```suggestion
- Run a basic speedtest:
```

File: pages/linux/speedtest.md
Comment (bl-ue) [inline pages/linux/speedtest.md:13]:
```suggestion
- Use HTTPS instead of HTTP:
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
Looks good @marchersimon, however there's no link.  Will you please add a link in the form:
```
> Test internt bandwith using speedtest.net
> More information: <url here (https if possible)>.`
```

**Comment by navarroaxel** [comment]
Hi @marchersimon we have a `speedtest-cli` in https://github.com/tldr-pages/tldr/blob/master/pages/common/speedtest-cli.md maybe we need an alias because the package install both commands `speedtest` and `speedtest-cli`.

**Comment by bl-ue** [comment]
@navarroaxel are you sure it's the same one?

**Comment by navarroaxel** [comment]
yep, but I didn't mean to close this PR. I think we need an alias page because I think a few people use `speedtest-cli`

**Comment by bl-ue** [comment]
We also need an alias system in general 🙏🏻

**Comment by marchersimon** [comment]
> yep, but I didn't mean to close this PR. I think we need an alias page because I think a few people use `speedtest-cli`

How can I do that? I didn't find anything about alias pages in `CONTRIBUTING.md`

**Review by einverne — APPROVED** [review]
Looks good to me, thanks for the new page.

**Review by navarroaxel — APPROVED** [review]

**Comment by bl-ue** [comment]
I already have a PR queued for speedtest-cli.  I vote for: merge this PR as is and then after #5284 is merged, sync that and this page.  @navarroaxel @einverne what do you think?

**Comment by einverne** [comment]
> I already have a PR queued for speedtest-cli.  I vote for: merge this PR as is and then after #5284 is merged, sync that and this page.  @navarroaxel @einverne what do you think?

It's ok with me, but we seems to need an alias mechanism.

**Comment by bl-ue** [comment]
Totally, but that's for a different day unfortunately.  Ideally we would use YAML frontmatter and place 
```markdown
---
title: speedtest-cli
alias: speedtest
---
```
as the only content in `speedtest-cli.md`.  Someday we shall do it, because everyone uses frontmatter and it could solve a lot of problems for us, but...

**Review by bl-ue — APPROVED** [review]

**Comment by bl-ue** [comment]
Oops! this was in `linux/`.  I'll move it to `common/` in this experimental PR that I'm opening.

**Pull request merged**
